### ISSUE TITLE
Fix request: AvoidSpringMVCMemoryLeaks false positive on non MVC code

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#501]
```java
@Component
public final class Foo {

    @RequestMapping(value = "/customers", params = {"title"}, method = RequestMethod.GET)
    @ResponseBody
    public Customer getCustomer(@RequestParam String title) {
        Id id = getCurrentUser().getId();
        return checkNotNull(service.findCustomer(id, title), "Customer with title [" + title + "] is not found"); // <- false positive
    }
    
}
```
